### DISCUSSION TITLE
chrome-extension-metamask

### DISCUSSION THREAD

**Discussion by eFutureDev** [community/community#45815] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Currently I am trying to build app with metamask extensions.
`const { BrowserWindow, app } = require('electron')
BrowserWindow.addExtension('/path/to/extension')`
Above is the main code and there was no compile errors. But in QA, it shows it doesn't support metamask.

**Comment by SuperHeroDev** [comment, marked as answer]
I implemented metamask extensions on electron app correctly.
here is the details process for metamask + Electron app.
https://medium.com/swapynetwork/integrating-metamask-with-electron-a-simple-secure-and-non-intrusive-approach-517a04da1656

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @eFutureDev ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
